.github/workflows/test.yml

```diff
@@ -119,3 +120,89 @@ jobs:
         uses: actions/checkout@v2
       - run: brew install proj
       - run: cargo test ${{ matrix.features }}
+
+  proj-sys-windows:
+    name: proj-sys windows
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: windows-latest
+    env:
+      _PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC: 0
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {
+              target: "x86_64-pc-windows-msvc",
+              VCPKG_DEFAULT_TRIPLET: "x64-windows-static",
+              RUSTFLAGS: "-Ctarget-feature=+crt-static",
+              features: "",
+            }
+          - {
+              target: "x86_64-pc-windows-msvc",
+              VCPKG_DEFAULT_TRIPLET: "x64-windows-static-md",
+              features: "",
+            }
+          - {
+              target: "x86_64-pc-windows-msvc",
+              VCPKG_DEFAULT_TRIPLET: "x64-windows",
+              VCPKGRS_DYNAMIC: 1,
+              # "cargo test --doc" broken with dynamic lib on CI machine (missing dll)
+              features: "--all-targets",
+            }
+          # - {
+              # target: "i686-pc-windows-msvc",
+              # VCPKG_DEFAULT_TRIPLET: "x86-windows-static",
+              # RUSTFLAGS: "-Ctarget-feature=+crt-static",
+              # features: "",
+            # }
+          # - {
+              # target: "i686-pc-windows-msvc",
+              # VCPKG_DEFAULT_TRIPLET: "x86-windows-static-md",
+              # features: "",
+            # }
+          # - {
+              # target: "i686-pc-windows-msvc",
+              # VCPKG_DEFAULT_TRIPLET: "x86-windows",
+              # VCPKGRS_DYNAMIC: 1,
+              # features: "--all-targets",
+            # }
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install vcpkg
+        run: |
+          git clone https://github.com/Microsoft/vcpkg.git vcp
+          # PROJ version 7.2.1 - https://github.com/microsoft/vcpkg/search?q=proj4&type=commits
```

Comment:
So when we update libproj, we'll need to see what version of vcpkg introduced support for the new libproj, and update this sha accordingly?

Reply:
It is not mandatory, we can remove the checkout line if I find how to check MINIMUM_PROJ_VERSION